broadcast/index.md

```diff
@@ -20,12 +20,8 @@ interesting developments in the ecosystem around Trino.
 ## Upcoming episodes
 
 <dl>
-<dt>31 October 2024: Trino Community Broadcast 67 - Extra speed with Exasol and Trino</dt>
-<dd>
-<a href="https://www.linkedin.com/in/thomas-bestfleisch/">Thomas Bestfleisch</a>
-from <a href="{{site.url}}/ecosystem/data-source.html#exasol">Exasol</a> join us to
-talk about advantages and scenarios of using the in-memory, column-oriented RDBMS Exasol with Trino.</dd>
-
+<dt>21 November 2024: Trino Community Broadcast 68</dt>
```

Comment:
fyi @colebow @bitsondatadev - I am not available for 14th .. so going for 21st .. topic yet to be determined ..